reana_workflow_controller/workflow_run_manager.py

```diff
@@ -357,7 +357,7 @@ def _create_job_spec(self, name, command=None, image=None,
             get_env_secrets_as_k8s_spec()
 
         user = \
-            secrets_store.get_secret_value('HTCONDORCERN_USERNAME') or \
+            secrets_store.get_secret_value('CERN_USER') or \
```

Comment:
Please amend the commit log message to say `CERN_USER` also there... otherwise people could misunderstand...